### PR TITLE
Add Contract name constants and bump to 0.13

### PR DESCRIPTION
**Pull request by felix2feng**

### CODE REVIEW

File: deployments/contractNames.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  RebalancingTokenIssuanceModule: 'RebalancingTokenIssuanceModule',
+  SetTokenFactory: 'SetTokenFactory',
+  StandardFailAuctionLibrary: 'StandardFailAuctionLibrary',
+  StandardPlaceBidLibrary: 'StandardPlaceBidLibrary',
+  StandardProposeLibrary: 'StandardProposeLibrary',
+  StandardSettleRebalanceLibrary: 'StandardSettleRebalanceLibrary',
+  StandardStartRebalanceLibrary: 'StandardStartRebalanceLibrary',
+  TransferProxy: 'TransferProxy',
+  Vault: 'Vault',
+  WhiteList: 'WhiteList'  ,
```
Comment (asoong) [inline deployments/contractNames.ts]: white space

File: deployments/outputs.json [hunk 2]
```diff
@@ -36,5 +36,45 @@
       "last_deployment_stage": 5,
       "network_id": 42
     }
+  },
+  "50-development": {
```
Comment (asoong) [inline deployments/outputs.json]: unstage these in source tree before committing

File: deployments/outputs.ts [hunk 3]
```diff
@@ -37,5 +37,45 @@ export const outputs =
       "last_deployment_stage": 5,
       "network_id": 42
     }
+  },
+  "50-development": {
```
Comment (asoong) [inline deployments/outputs.ts]: same

File: deployments/contractNames.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
+export const CONTRACT = {
+  BitEthInitialCollateralSet: 'BitEthInitialCollateralSet',
```
Comment (asoong) [inline deployments/contractNames.ts]: This seems really roundabout and error prone. Can we do something like: https://stackoverflow.com/questions/10314338/get-name-of-object-or-class

Import them all from the `./ts` folder and then use this?

File: deployments/outputs.json (hunk 2 above)
Comment (kermankohli) [reply deployments/outputs.json]: if you run `yarn delete-developments` it should wipe it

**Pull request merged**